gocourse05/go.mod

```
@@ -0,0 +1,3 @@
+module gocourse05
```

Можна попросити тебе перейменувати пул реквест?

Зараз "Lets go4", але ж це домашка до вебінару 5.

Тому пропоную назвати "Homework 5".

---

Модулі краще називати ось так:
```suggestion
module github.com/b1rr0/go-practice/gocourse05
```

Почитати тут https://go.dev/ref/mod#module-path